### ISSUE TITLE
Android - Search Results Screen - Color Contrast of Search Box border

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#23]
The color contrast of the border around the search box ( #999CA2 ) is not enough against the white background ( #ffffff ).

The color contrast ratio is 2.75:1 and needs to be 4,5:1 or higher to pass.

Please see attached screenshot.

![android - search results screen - poor color contrast of search box and results border](https://user-images.githubusercontent.com/10423487/57148654-0f42bf80-6d98-11e9-9a25-82739a40c08a.png)

**Comment by KyleLivingstonDOL** [comment]
Thanks Josh. We will take a look into this.

**Comment by jfc3-dol** [comment]
You're welcome and it's John not Josh.

**Comment by KyleLivingstonDOL** [comment]
Thanks John,

So if we increase the contrast -- by making the borders a darker blue, for example -- as long as it meets the 4.5:1, then it is fine?

**Comment by jfc3-dol** [comment]
By making the borders, a darker blue could allow them to pass color contrast depending on the blue.

I use Jonathan Snook's online Colour Contrast Checker ( https://snook.ca/technical/colour_contrast/colour.html ) to make sure the color contrast ratio is 4.5:1, and both of the WCAG 2.0 AA boxes say "YES". All you have to do is put the text and background colors in the foreground and background fields, and it will tell you what you need to know.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

If we update the boxes in black, would that suffice to meet the color contrast requirements?

Thanks,
Kyle

**Comment by jfc3-dol** [comment]
Yes, you could make them black on the white background to pass, or you could use a dark charcoal gray ( #3d3d3d ) on a white ( #ffffff ) background which gives you enough color contrast but is not as jarring on the eyes as using black.

https://snook.ca/technical/colour_contrast/colour.html#fg=3D3D3D,bg=FFFFFF

**Comment by KyleLivingstonDOL** [comment]
> Yes, you could make them black on the white background to pass, or you could use a dark charcoal gray ( #3d3d3d ) on a white ( #ffffff ) background which gives you enough color contrast but is not as jarring on the eyes as using black.
> 
> https://snook.ca/technical/colour_contrast/colour.html#fg=3D3D3D,bg=FFFFFF

Thanks John. We will do that. We will make the borders a dark charcoal gray.

**Comment by jfc3-dol** [comment]
Why is the top border to the search box a medium gray  ( #919191 ) while the other three sides are set to #0C0C0C. So the color contrast ratio for the top is 3.18:1 and the other three sides are 19.56:1.

Please make all four search box borders #0C0C0C so it passes.

Attached is a screenshot.

![comply chain - search box top boder does not have enough color contrast](https://user-images.githubusercontent.com/10423487/59310048-303de080-8c73-11e9-92b1-0d9581e0e0b7.png)

**Comment by KyleLivingstonDOL** [comment]
ZTI, can you please address?

**Comment by LeHaine** [comment]
Not able to reproduce this issue. It does use a single solid border of #3d3d3d.

![image](https://user-images.githubusercontent.com/4951150/59445170-262ff500-8dcd-11e9-9cb7-79111f3bc6b3.png)

**Comment by jfc3-dol** [comment]
Okay, I figured it was one solid border but wanted to check since my screen or the screenshot made the top border look lighter.

Closing the issue.
